src/app/games/lasca/tests/LascaControlAndDominationMinimax.spec.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable max-lines-per-function */
+import { Player } from 'src/app/jscaip/Player';
+import { RulesUtils } from 'src/app/jscaip/tests/RulesUtils.spec';
+import { MGPOptional } from 'src/app/utils/MGPOptional';
+import { LascaControlAndDominationMinimax } from '../LascaControlAndDomination';
+import { LascaRules } from '../LascaRules';
+import { LascaPiece, LascaSpace, LascaState } from '../LascaState';
+
+describe('LascaControlAndDominateMinimax', () => {
+
+    const O: LascaSpace = new LascaSpace([LascaPiece.ZERO]);
+    const X: LascaSpace = new LascaSpace([LascaPiece.ONE]);
+    const _: LascaSpace = LascaSpace.EMPTY;
+    let minimax: LascaControlAndDominationMinimax;
+
+    beforeEach(() => {
+        const ruler: LascaRules = LascaRules.get();
+        minimax = new LascaControlAndDominationMinimax(ruler, 'Lasca Control And Dominate Minimax');
+    });
+    it('should not count the immobilized stacks', () => {
+        // Given two board with the exact same stacks, one having blocked stacks
+        const immobilizedState: LascaState = LascaState.from([
+            [X, _, _, _, _, _, _],
+            [_, O, _, _, _, _, _],
+            [_, _, O, _, _, _, _],
+            [_, _, _, _, _, _, _],
+            [_, _, _, _, X, _, _],
+            [_, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _],
+        ], 0).get();
+        const mobileState: LascaState = LascaState.from([
+            [X, _, _, _, _, _, _],
+            [_, _, _, O, _, _, _],
+            [_, _, O, _, _, _, _],
+            [_, _, _, _, _, _, _],
+            [_, _, _, _, X, _, _],
+            [_, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _],
+        ], 0).get();
+
+        // When comparing them
+        // Then the one with blocked stack should be considered lesser
```

Comment:
less good? weaker?

Reply:
Then the one with mobile stacks should be considered better ?

Reply:
oui

Reply:
j'ai changé celui dont on parle pour garder better.
Ton poil rectal !